cmd/release-controller/sync_release.go

```diff
@@ -384,7 +384,7 @@ func (c *Controller) ensureReleaseMirrorJob(release *releasecontroller.Release,
 			cliImage = release.Config.OverrideCLIImage
 		}
 
-		job, prefix := newReleaseJobBase(name, cliImage, release.Config.PullSecretName)
+		job, prefix := newReleaseJobBase(name, cliImage, release.Config.AlternateImageRepositorySecretName)
```

Comment:
This change will need to be rolled out in the correct order and not with everything.  I.e it could be disasterous if `release.Config.AlternateImageRepositorySecretName` isn't set before we roll the code out.